src/Resources/config/fixtures/advanced_promotion.yaml

```diff
@@ -0,0 +1,9 @@
+sylius_fixtures:
+    suites:
+        default:
+            fixtures:
+                monsieurbiz_advanced_promotion:
+                    options:
+                        promotion_advanced_configuration:
+                            -   code: 'new_year'
+                                after_tax: true
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏ Shouldn't the fixture be added in the `dist` folder ?

Reply:
No default suite fixtures are in plugins.